AprilTagTrackers/Helpers.cpp

```diff
@@ -180,7 +180,7 @@ cv::Mat getSpaceCalibEuler(cv::Vec3d rvec, cv::Vec3d tvec, double xOffset, doubl
 }
 
 // Calculates rotation matrix given euler angles.
-cv::Mat eulerAnglesToRotationMatrix(cv::Vec3f& theta)
+cv::Mat eulerAnglesToRotationMatrix(cv::Vec3f theta)

```

Comment:
Did you get a warning here? Try using a const ref instead.
```suggestion
cv::Mat eulerAnglesToRotationMatrix(const cv::Vec3f& theta)

```

Reply:
I don't recall having such, but I'll try your suggested change!

Reply:
Doing so in-fact caused an error.